expected_output:
  - The crashpod or might mention the pod 'crashpod-9688789bc-lgjzj'. Mentioning one of the reasons for a CrashLoopBackOff state. 1. due to command 'wget' exiting with code 125 2. having recursive calls 3. StackOverflowError in the application.
  - workload_healthy flag should be false
before_test: |
  kubectl apply -f https://gist.githubusercontent.com/robusta-lab/283609047306dc1f05cf59806ade30b6/raw
  sleep 60
after_test: |
  kubectl delete -f https://gist.githubusercontent.com/robusta-lab/283609047306dc1f05cf59806ade30b6/raw
evaluation:
  correctness: 1
# Success rate 100% for 100 evals
